src/mon/Monitor.cc

```diff
@@ -1118,7 +1118,7 @@ void Monitor::bootstrap()
   if (monmap->min_mon_release &&
       monmap->min_mon_release + 2 < (int)ceph_release()) {
     derr << "current monmap has min_mon_release "
-	 << ceph_release() << " (" << ceph_release_name(monmap->min_mon_release)
+	 << monmap->min_mon_release << " (" << ceph_release_name(monmap->min_mon_release)
```

Comment:
see also #27107, and the same comment also applies. i am quoting it here

> can we cast it to another integer type, unsigned for instance, for printing it? as uint8_t is always defined as an alias of unsigned char, if we just send it to ostream, it will be interpreted as an ASCII char, i think.